assets/src/dashboard/app/serializers/stories.js

```diff
@@ -19,7 +19,7 @@
  */
 import { migrate, DATA_VERSION } from '@web-stories-wp/migration';
 
-export default function reshapeStoryObject(editStoryURL) {
+export default function reshapeStoryObject() {
   return function (originalStoryData) {
```

Comment:
Now this doesn't need to be a function that returns another function anymore

```suggestion
export default function reshapeStoryObject(originalStoryData) {
```